src/components/Avatar/Avatar.react.js

```diff
@@ -0,0 +1,65 @@
+// @flow
+
+import * as React from "react";
+import { Icon } from "../";
+import cn from "classnames";
+import AvatarList from "./AvatarList.react";
+
+type Props = {|
+  +children?: React.Node,
+  +className?: string,
+  +image?: string,
+  +style?: Object,
+  +size?: "" | "sm" | "md" | "lg" | "xl" | "xxl",
+  +status?: "grey" | "red" | "yellow" | "green",
+  +placeholder?: boolean,
+  +icon?: string,
+  +color?: string,
+|};
+
+function Avatar({
+  className,
+  children,
+  image,
+  style,
+  size = "",
+  status,
+  placeholder,
+  icon,
+  color = "",
+  ...rest
+}: Props): React.Node {
+  const classes = cn(
+    {
+      avatar: true,
+      [`avatar-${size}`]: !!size,
```

Comment:
This is where the size prop is used in a computed property so it can't be null.
Could a better solution be doing `[`avatar-${size || ''}`]: !!size,` here and then we can remove the empty string from the size type definition?